api/controllers/StorageController.js

```diff
@@ -95,6 +96,26 @@ module.exports = {
             port: machine.plazaport
           }, req.allParams().path || 'C:\\');
         });
+    } else if (req.allParams().teams === 'true') {
+      getFiles = Team.findOne(req.user.team)
+        .then((teams) => {
+          return ConfigService.get('teamStorageAddress', 'teamStoragePort')
+            .then((config) => {
+              let teamStorageURL = 'http://' +
+                config.teamStorageAddress + ':' +
+                config.teamStoragePort + '/files?path=' +
+                encodeURI('/home/' + teams.username);
```

Comment:
What if `teams` if undefined (a call to `/apis/files` is done even though the user does not belong to a team ?